Classes/Dojo/Dojo_main.py

```diff
@@ -0,0 +1,223 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+# Cilass Dojo
+
+from Person.person_new import Person
+from Person.Staff.staff import Staff
+from rooms.room_new import Room
+import random
+import collections
+
+
+class Dojo(object):
+    def __init__(self):
+        self.list_persons = []
```

Comment:
Use class variables for data structures. This structures will be initialized every time class Dojo runs.